modules/default/newsfeed/newsfeed.js

```diff
@@ -273,5 +288,63 @@ Module.register("newsfeed",{
 		return string.charAt(0).toUpperCase() + string.slice(1);
 	},
 
+	notificationReceived: function(notification, payload, sender) {
+		Log.info(this.name + " - received notification: " + notification);
+		if(notification == "ARTICLE_NEXT"){
+			var before = this.activeItem;
+			this.activeItem++;
+			if (this.activeItem >= this.newsItems.length) {
+				this.activeItem = 0;
+			}
+			this.config.showDescription = false;
```

Comment:
the following lines repeat also for article previous and article less details, how about a reaset helper function which does it?